sdk/test-utils/recorder-new/src/utils/sessionFilePath.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { env, generateTestRecordingFilePath } from "@azure-tools/test-recorder";
+import { isNode } from "@azure/test-utils";
+import { relativeRecordingsPathForNode } from "./relativePathCalculator";
+
+export function sessionFilePath(testContext: Mocha.Test) {
+  if (!isNode) {
```

Comment:
instead of `isNode`, maybe have node and browser versions in different files?

Reply:
> but I remember we want to move away from isNode in libraries

This was for the things we use the rollup tree shaking for I think.
We don't rely on tree-shaking here, doesn't matter if it is used or not, so I don't mind this snippet/code being present in both browser and node or excluded in either with tree-shaking.

For example, I have added `relativePathCalculator.browser.ts` for `relativePathCalculator.ts` since I would need fs and path libraries, which is not the case here.